### PR TITLE
SUIT-9706 Prevent closing connection with JS API in case of debugging

### PR DESCRIPTION
**Pull request by givehug**
Make enableDebugMode request after session opening when running in debug mode

### CODE REVIEW

File: lib/testLauncher/SuitestLauncher.js [hunk 1]
```diff
@@ -127,7 +127,10 @@ class SuitestLauncher {
 		validateInput('INTERACTIVE', this.ownArgv);
 
 		try {
-			const {username, password, orgId, deviceId, appConfigId, inspect, inspectBrk, logDir} = this.ownArgv;
+			const {
+				username, password, orgId, deviceId, appConfigId, logDir,
+				inspect, inspectBrk, debugBrk,
```
Comment (andriichumak) [inline lib/testLauncher/SuitestLauncher.js]: We do not support debugBrk, it's deprecated

File: lib/testLauncher/SuitestLauncher.js [hunk 2]
```diff
@@ -145,6 +148,7 @@ class SuitestLauncher {
 							[envVars.SUITEST_SESSION_TOKEN]: deviceAccessToken,
 							[envVars.SUITEST_DEVICE_ID]: deviceId,
 							[envVars.SUITEST_APP_CONFIG_ID]: appConfigId,
+							[envVars.SUITEST_DEBUG_MODE]: (inspectBrk || inspect || debugBrk) ? 'yes' : 'no',
```
Comment (andriichumak) [inline lib/testLauncher/SuitestLauncher.js]: This condition repeats on line 156. Perhaps it makes sense to make a variable for this.

File: lib/testLauncher/SuitestLauncher.js (hunk 1 above)
Comment (givehug) [reply lib/testLauncher/SuitestLauncher.js]: Ok, we should change docs description about vs-code dbugging

Comment (andriichumak) [reply lib/testLauncher/SuitestLauncher.js]: I'll create a ticket in Jira for this

**Comment by givehug** [comment]
@andriichumak updated

**Comment by andriichumak** [comment]
@givehug please resolve conflicts

**Comment by givehug** [comment]
@andriichumak  merged

**Pull request merged**